_rules/explicit-SVG-image-has-name-7d6734.md

```diff
@@ -0,0 +1,167 @@
+---
+id: 7d6734
+name: Explicit SVG image has accessible name
+rule_type: atomic
+description: |
+   This rule checks that each SVG image element that is explicitly included in the accessibility tree has an accessible name.
+accessibility_requirements:
+  wcag20:1.1.1: # Non-Text Content (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+authors:
+  - Bryn Anderson
+---
+
+## Applicability
+
+The rule applies to any element in the [SVG](https://www.w3.org/2000/svg) namespace with an explicit [semantic role](#semantic-role) of either `img`, `graphics-document`, `graphics-symbol`, that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+**Note**: The [SVG Accessibility API Mappings](https://www.w3.org/TR/svg-aam-1.0/#include_elements) specifies that many elements in the SVG namespace are purely presentational and should not be included in the accessibility tree unless indicated otherwise through the use of text alternative content, an explicit WAI ARIA role, or a valid `tabindex` attribute.
+
+## Expectation
+
+Each target element has an [accessible name](#accessible-name) that is not only [whitespace](#whitespace).
+
+## Assumptions
+
+This rule assumes that the presence of one of the roles outlined in the applicability indicates the authors intent to include the element in the accessibility tree and thus convey information to the user about that element.
+
+## Accessibility Support
+
+The [HTML Accessibility API Mappings](https://www.w3.org/TR/html-aam-1.0/#html-element-role-mappings) specify that the `<svg>` element has an implicit role of `graphics-document`. However browser support for the `graphics-document` role and the [SVG Accessibility API Mappings (work in progress)](https://www.w3.org/TR/svg-aam-1.0) is inconsistent.
+
+This rule is limited to the implicit use of roles, as a clear indication that content should convey meaning, until the [SVG Accessibility API Mappings (work in progress)](https://www.w3.org/TR/svg-aam-1.0) is more stable and borwser support is more consistent.
```

Comment:
borwser > browser
More importantly, I do not really understand what you mean in this paragraph (unless you meant "explicit use of roles" instead of "implicit use of roles)

Reply:
changed implicit > explicit